### PR TITLE
Creating spaces

### PR DESCRIPTION
**Pull request by dhopz**
We have added functionality for created a space and listing all spaces

### CODE REVIEW

File: spec/spaces_spec.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+require 'spaces'
+
+describe 'spaces' do
+
+    it '.create' do
+        connection = PG.connect(dbname: 'bnbapp', user: 'postgres', password: 'password')
+        connection.exec("INSERT INTO spaces (space_name, space_description, price_per_night) VALUES ('A new Space', 'this is the description', '100');")
+
+        Spaces.create((name,description,price_per_night)
```
Comment (mandyvuong) [inline spec/spaces_spec.rb]: typo extra (

File: lib/spaces.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+require 'pg'
+#require_relative 'database_connection'
+
+class Spaces
+
+    def self.create(name:, description:, price_per_night:)
+        connection = PG.connect(dbname: 'bnbapp', user: 'postgres', password: 'password')
```
Comment (lildann) [inline lib/spaces.rb]: might be some conflict here - we have named our databases 'bnb' and 'bnb_test'

File: lib/spaces.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    def self.create(name:, description:, price_per_night:)
+        connection = PG.connect(dbname: 'bnbapp', user: 'postgres', password: 'password')
+        connection.exec("INSERT INTO spaces (space_name, space_description, price_per_night) VALUES ('#{name}', '#{description}', '#{price_per_night}');")
+    end
+
+    def self.all
+        connection = PG.connect(dbname: 'bnbapp', user: 'postgres', password: 'password')
+        result = connection.exec("SELECT * FROM spaces;")
+
+        result.values
```
Comment (lildann) [inline lib/spaces.rb:16]: This is neat!

File: app.rb [hunk 4]
```diff
@@ -10,5 +11,25 @@ class Bnb < Sinatra::Base
     'Hello World'
   end
```
Comment (lildann) [inline app.rb:12]: Potential conflict with out root '/' - remove?

**Review by lildann — COMMENTED** [review]
Nice one. We should discuss database connection as we've approached this differently in our branches.

**Pull request merged**